projects/kit/components/toggle/toggle.template.html

```diff
@@ -7,33 +7,52 @@
     [invalid]="computedInvalid"
 >
     <div class="toggle">
-        <tui-loader
-            *ngIf="showLoader"
-            class="loader"
-            [size]="loaderSize"
-            [inheritColor]="true"
-            [showLoader]="checked"
-        ></tui-loader>
-        <tui-svg
-            *ngIf="showIcons && !showLoader"
-            class="icon"
-            automation-id="tui-toggle__check-icon"
-            [src]="iconOn"
-        ></tui-svg>
+        <span
+            polymorpheus-outlet
```

Comment:
Right now custom content makes it impossible to use `showLoader`. This should only be applied to the `tui-svg` with icon below.

Reply:
Ops, my fault. I've corrected )